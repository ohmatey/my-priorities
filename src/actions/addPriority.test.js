import { addPriorityById } from './'

describe('addPriorityById function', () => {
  it('matches snapshot with no id', () => {
    const addedPriorityWithNoId = addPriorityById()

    expect(addedPriorityWithNoId).toMatchSnapshot()
  })

  it('matches snapshot with id', () => {
    const addPriorityActionWithId = addPriorityById(1)

    expect(addPriorityActionWithId).toMatchSnapshot()
  })
})
