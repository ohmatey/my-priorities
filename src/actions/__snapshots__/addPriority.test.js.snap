// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addPriorityById function matches snapshot with id 1`] = `
Object {
  "id": 1,
  "type": "ADD_PRIORITY",
}
`;

exports[`addPriorityById function matches snapshot with no id 1`] = `
Object {
  "id": undefined,
  "type": "ADD_PRIORITY",
}
`;
